initramfs-init.in

```diff
@@ -279,6 +279,23 @@ is_url() {
 	esac
 }
 
+# Do some tasks to make sure mounting the ZFS pool is A-OK
+prepare_zfs_root() {
+	local _root_vol=${KOPT_root#ZFS=}
+	local _root_pool=${_rool_vol%%/*}
+
+	# Force import if this has been imported on a different system previously
+	if [ "$KOPT_zfs_force" = 1 ]; then
+		zpool import -f $_root_vol
```

Comment:
I think you should import with -N here, otherwise every unencrypted dataset with mountpoint set will be mounted automatically on import

Reply:
Oh right, openrc should mount those. Thanks